sequencer/src/bin/cdn-broker.rs

```diff
@@ -80,15 +79,13 @@ struct Args {
     key_seed: u64,
 }
 
-#[cfg_attr(async_executor_impl = "tokio", tokio::main(flavor = "multi_thread"))]
-#[cfg_attr(async_executor_impl = "async-std", async_std::main)]
+#[async_std::main]
 async fn main() -> Result<()> {
     // Parse command line arguments
     let args = Args::parse();
 
     // Initialize tracing
-    setup_logging();
-    setup_backtrace();
+    tracing_subscriber::fmt::init();
```

Comment:
why?

Reply:
It wasn't printing any logs to `stdout` until I changed this. I can investigate it further if needed

Reply:
not a big deal. Just weird because I think we use the `setup_logging()` thing in other binaries. We can keep it this way though